docs/source/models/gbd2017/causes/tb_hiv_model_vv.rst

```diff
@@ -0,0 +1,87 @@
+.. _cause_model_validation:
+
+========================================
+TB/HIV model validation and verification
+========================================
+
+Obejective
+----------
+External
+	- Model results should be checked against local TB epidemiology
+		- e.g. rate of decline in burden of disease should be compared
+		  with historical evidence.
+	- Compare our results (e.g. ICERs) to similar models or empirical
+	  assessments where possible
+Internal
+	- Calibrate simulation baseline against GBD 2017 results
+		- Sim outputs mean should perfectly match GBD results
+		- Sim outputs uncertainty should wider than GBD results
+			- Stochastic + individual heterogeneity

```

Comment:
bullet is unclear, but I think you mean "The uncertainty intervals of sim outputs should be wider than GBD results, because they include parameter uncertainty *and* stochastic uncertainty".  (Heterogeneity should not increase the uncertainty interval, I think.)

Reply:
Thanks for clarifying it!